modules/space/rotation/spicerotation.cpp

```diff
@@ -111,16 +128,23 @@ SpiceRotation::SpiceRotation(const ghoul::Dictionary& dictionary)
 
     _sourceFrame.onChange([this]() { requireUpdate(); });
     _destinationFrame.onChange([this]() { requireUpdate(); });
+    _fixedDate.onChange([this]() {
```

Comment:
If you place the assignment to _fixedDate after the `onChange` you don't need to repeat the `ephemerisTimeFromDate` line as the `onChange` will be triggered by that assignment